roles/openshift_health_checker/openshift_checks/logging/curator.py

```diff
@@ -12,12 +12,10 @@ class Curator(LoggingCheck):
     name = "curator"
     tags = ["health", "logging"]
 
-    logging_namespace = None
-
     def run(self, tmp, task_vars):
         """Check various things and gather errors. Returns: result as hash"""
 
-        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        super(Curator, self).logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
```

Comment:
```python
self.logging_namespace
```

Python does the attribute look up, and then we want to change the instance attribute, not the parent class' attribute.

In an interpreter run time where two instances share the same parent class, we don't want the `run` method of one instance to affect the other.